menu.html

```diff
@@ -1,57 +1,86 @@
 <!DOCTYPE html>
 <html>
+
+<head>
+    <title>PizzaParty</title>
+    <link href="css/style.css" rel="stylesheet">
+</head>
 <meta charset="UTF-8">
 <title>Menu</title>
```

Comment:
these lines _do_ need to be inside of `<head>` tag

Reply:
this is fixed